packages/devtools/src/commands/findElement.js

```diff
@@ -0,0 +1,35 @@
+import findElementByXPath from '../scripts/findElementByXPath'
+import { SUPPORTED_SELECTOR_STRATEGIES, SERIALIZE_PROPERTY, SERIALIZE_SELECTOR } from '../constants'
+import { findElement as findElementUtil } from '../utils'
+import cleanUp from '../scripts/cleanUpSerializationSelector'
+
+export default async function findElement ({ using, value }) {
+    if (!SUPPORTED_SELECTOR_STRATEGIES.includes(using)) {
+        throw new Error(`selector strategy "${using}" is not yet supported`)
+    }
+
+    const page = this.windows.get(this.currentWindowHandle)
+    let needsCleanUp = false
+
+    if (using === 'xpath') {
+        const foundElement = await page.$eval('html', findElementByXPath, value, null, SERIALIZE_PROPERTY)
```

Comment:
nothing is done with the element found?

Reply:
nope, the way it works is that we use `$eval` to run some JS that finds elements using an xPath. All these elements get a data attribute so that they can be found using CSS selectors in a secondary call. At the end we clean up the data attributes and return the value.

Reply:
`foundElement` is a boolean in this case

Reply:
I will add a comment